conda/recipes/meta.yaml

```diff
@@ -18,17 +16,18 @@ source:
 
 build:
   number: {{ git_revision_count }}
+  noarch: python
   script: python -m pip install . -vvv
 
-
 requirements:
+  build:
+    - python
+    - setuptools
   run:
-  {% for req in setup_py_data.get('install_requires', []) %}
-    - {{ req }}
-  {% endfor %}
     - python
-    - tensorflow
```

Comment:
Removed tensorflow and nvtabular deps from this conda package config so that the install matches the pip version with these as optioinal dependencies